tests/unit/h/views/admin/users_test.py

```diff
@@ -75,19 +76,6 @@ def test_users_index_strips_spaces(models, pyramid_request):
     models.User.get_by_username.assert_called_with(pyramid_request.db, "bob", "foo.org")
 
 
-@users_index_fixtures
```

Comment:
Merged some tests in this file, they were one-assertion-per-test style which becomes verbose pretty quickly. Done in a separate commit.